### PR TITLE
created home page for mango site

### PR DESCRIPTION
**Pull request by r-susmitha**
created home page for mango site

### CODE REVIEW

**Comment by r-susmitha** [comment]
check the home site of mango

File: index.html [hunk 1]
```diff
@@ -1,9 +1,32 @@
...
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width,initial-scale=1.0">
+        <title>Mango Site</title>
+        <link rel="stylesheet" href="style.css">
+
+    </head>
+    <body>
+        <div class="title">
+            <h1>PENNY JUICE</h1>
```
Comment (visprabu) [inline index.html:12]: Not aligned properly. Try to align it as displayed in the https://dribbble.com/shots/20735239-Juice-website-landing-page

File: index.html [hunk 2]
```diff
@@ -1,9 +1,32 @@
...
+            <h3>IT MAKES CENTS</h3>
+        </div>
+        <div class="nav-bar">
```
Comment (visprabu) [inline index.html:15]: When hovered over the each element in the list, using css add a line under the text. Ex: https://codepen.io/MauriciAbad/pen/QJmwOY

File: index.html [hunk 3]
```diff
@@ -1,9 +1,32 @@
...
+            <ul>
+                <li><a href="Home">Home</a></li>
+                <li><a href="ChildCarejuice">ChildCare Juice</a></li>
+                <li><a href="Order Now">Order Now</a></li>
```
Comment (visprabu) [inline index.html:19]:
```suggestion
                <li><a href="#">Order Now</a></li>
```
For all the elements in the li

File: index.html [hunk 4]
```diff
@@ -1,9 +1,32 @@
...
+            </ul>
+            <button>Contact</button>
+        </div>
+        <div class="container">
+            <img src="resources/images/fruits-1.jpg"   alt="apple"/>         
+            <h1>Best Juice<br> For Child's</h1>
```
Comment (visprabu) [inline index.html:25]: Increase the font size to match the image

File: index.html [hunk 5]
```diff
@@ -1,9 +1,32 @@
...
+        <div class="nav-bar">
+            <ul>
+                <li><a href="Home">Home</a></li>
+                <li><a href="ChildCarejuice">ChildCare Juice</a></li>
+                <li><a href="Order Now">Order Now</a></li>
+            </ul>
+            <button>Contact</button>
+        </div>
+        <div class="container">
+            <img src="resources/images/fruits-1.jpg"   alt="apple"/>
```
Comment (visprabu) [inline index.html:24]: Use png image without background, ex: https://www.rawpixel.com/search/fruit%20png?page=1&path=_topics&sort=curated

File: index.html (hunk 1 above)
Comment (r-susmitha) [reply index.html:12]: changed the alignment

File: index.html (hunk 2 above)
Comment (r-susmitha) [reply index.html:15]: changed the nav-bar alignment with hover underline

File: index.html (hunk 3 above)
Comment (r-susmitha) [reply index.html]: separated with different div elements and alignment has changed and buttons with box shadow

File: index.html (hunk 5 above)
Comment (r-susmitha) [reply index.html]: used different img with blur effects

**Pull request closed without merging**